keeper/wormhole_core.go

```diff
@@ -96,19 +96,29 @@ func (k *Keeper) HandleCoreGovernancePacket(ctx context.Context, pkt types.Gover
 		if err != nil {
 			return errors.Wrap(err, "failed to get config from state")
 		}
-		blockTime := uint64(k.headerService.GetHeaderInfo(ctx).Time.Unix())
 
 		oldIndex := cfg.GuardianSetIndex
 		if index != oldIndex+1 {
 			return fmt.Errorf("invalid guardian set index: expected %d, got %d", oldIndex+1, index)
 		}
 
+		oldGuardianSet, err := k.GuardianSets.Get(ctx, oldIndex)
+		if err != nil {
+			return errors.Wrap(err, "failed to get old guardian set from state")
+		}
+		blockTime := uint64(k.headerService.GetHeaderInfo(ctx).Time.Unix())
+		oldGuardianSet.ExpirationTime = blockTime + 86400
```

Comment:
Should we make the 1 day in seconds, `86400`, as a constant value so it is easier for maintainers to understand what this value represents?